### DISCUSSION TITLE
eclipse question: changing project config files while the project is open

### DISCUSSION THREAD

**Discussion by PerennialNovice** [Sloeber/arduino-eclipse-plugin#1678] [General]
I know this is not sloeber related, but it seems there is a lot of eclipse/CDT knowledge here as well :)

I sometimes experience strange errors after pulling a commit from git.
We also have git monitor .project, .cproject, .sproject and sloeber.config

I suspect that this might not be a good idea, since the project config files are getting changed while the project is open in sloeber.

Can anyone confirm that this causes problems?

**Comment by jantje** [comment]
I know some people think it is a bad idea but I personally do not check in .* files.
I'm from the opinion : if it is hidden it should not be under version control.
That is also why there is a convert to sloeber project functionality.
When you share the files with people who have the same os but sloeber installed in a different location you will experience problems when sharing .* files.
When you share the files with people who have a different os you will experience problems when sharing .* files.
